ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -1292,23 +1295,34 @@ Rectangle {
 
                             onTextChanged: {
                                 if (length <= control.messageLimit) {
-                                    var symbols = ":='xX><0O;*dB8-D#%\\";
-                                    var symbolIdx = ((cursorPosition > 2) && (symbols.indexOf(getText((cursorPosition - 2), (cursorPosition - 1)))!==-1))
-                                                    ? (cursorPosition -1) : (symbols.indexOf(getText(0, 1))!==-1) ? 2 : -1;
-                                    if ((getText(symbolIdx-2, (symbolIdx-1)) === " ") || (symbolIdx === 2)) {
-                                        const emojis = StatusQUtils.Emoji.emojiJSON.emoji_json.filter(function (emoji) {
-                                            if (emoji.aliases_ascii.includes(getText((symbolIdx-1), (symbolIdx+1))) ||
-                                                emoji.aliases_ascii.includes(getText((symbolIdx-2), (symbolIdx+1)))) {
-                                                var has2Chars = emoji.aliases_ascii.includes(getText((symbolIdx-4), (symbolIdx-2)));
-                                                replaceWithEmoji("", getText((symbolIdx - (has2Chars ? 3 : 2)), symbolIdx), emoji.unicode);
-                                            }
-                                        })
-                                    }
                                     if (length === 0) {
                                         mentionsPos = [];
+                                    } else {
+                                        if (getText(cursorPosition, cursorPosition - 1) === " ") { // trigger inline emoji replacements after space
+                                            // figure out last word (between spaces), max length of 5
+                                            var lastWord = ""
+                                            const cursorPos = cursorPosition - 2 // just before the last non-space character
+                                            for (let i = cursorPos; i > cursorPos - 6; i--) { // go back until we found a space or start of line
```

Comment:
WDYT of extracting the emoji replacement to a function and use it on enter pressed also? (I know it wasn't in the scope of this PR to fix the emojis 😄 )

Currently the user cannot send only an emoji in the message (hitting enter instead of space)
Try `:b` + Enter

Reply:
Fixed, check it out! :)